docs/src/pages/demos/app-bar/app-bar.md

```diff
@@ -26,3 +26,7 @@ It can transform into a contextual action bar or used as a navbar.
 ## Dense (desktop only)
 
 {{"demo": "pages/demos/app-bar/DenseAppBar.js"}}
+
+## App Bar with search field
+
+{{"demo": "pages/demos/app-bar/SearchAppBar.js"}}
```

Comment:
I think this should be grouped with the other "App Bar with..." examples.

Reply:
corrected.